auctions/models.py

```diff
@@ -28,6 +28,8 @@
                          "client_secret": settings.PAYPAL_CLIENT_SECRET,
                          })
 
+paypay_recipient = settings.PAYPAL_PAYOUT_RECIPIENT
```

Comment:
typo: `paypay` also conventional to put these module-level constants in all caps: `PAYPAL_RECIPIENT`